src/parser/ts/index.ts

```diff
@@ -48,6 +64,32 @@ const NoTimeseriesFound = (filter: StringMap, expr: string) => {
   );
 };
 
+/**
+ * Injects default values of aggregate and granularity (if it is not provided).
+ * Mutates <query> properties array with default aggregate or granularity values.
```

Comment:
it doesn't mutate, it returns a copy

Reply:
well, internally it mutates, but it's not visible outside anyway

Reply:
I've mentioned it cause implementation is a bit step away of a functional approach

Reply:
I believe it just gives wrong assumptions to the user, as what matters is inputs/outputs, not internal logic.